traffic_portal/app/src/common/modules/form/deliveryServiceJob/form.deliveryServiceJob.tpl.html

```diff
@@ -31,7 +31,16 @@
         <div class="helptext">
             <aside class="warning">
                 <h6>Warning</h6>
-                <p>Delivery service origin(s) must support IMS in order to safely invalidate content</p>
+                <dl>
+                    <dt>Refresh (preferred default)</dt>
+                    <dd>Marks the matching asset(s) as "stale" and an IMS request of the asset(s) from the origin will be performed. Origins that fully adhere 
+                    to the <a href="https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.25" target="_blank" rel="external">If-Modified-Since</a> 
+                    and <a href="https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.26" target="_blank" rel="external">If-None-Match</a> specification can safely invalidate content using Refresh.</dd>
+                    <dt>Refetch</dt>
+                    <dd>Marks the matching asset(s) as a cache "miss" and a new full request of the asset(s) from the origin will be performed. 
+                    This operation is expensive and it is <em>highly recommended</em> that the specified regex is very specific when using Refetch.</dd>
+                    <p>For additional information please see the documentation for <a href="https://traffic-control-cdn.readthedocs.io/en/latest/overview/jobs.html" target="_blank" rel="external">Content Invalidation Jobs.</a></p>
```

Comment:
The only permitted content of a [`<dl>` element tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) is:

> Either: Zero or more groups each consisting of one or more `<dt>` elements followed by one or more `<dd>` elements, optionally intermixed with `<script>` and `<template>` elements.

i.e. `<p>` is invalid here. This paragraph doesn't appear associated with either term in specific, so it should probably go outside the `<dl>` altogether. However, if appropriate, `<p>` is "Flow Content" and therefore allowed within a `<dd>`.

Reply:
This was an oversight on my part. I'll get that out of the `<dl>` tag.